src/bot/admin.py

```diff
@@ -97,5 +105,5 @@ def __init__(self, bot):
             ))
 
 
-def setup(bot):
+def setup(bot):  # pylint: disable=missing-docstring
```

Comment:
I thought the same, but wouldn't this just result in 5 completely same docstrings for every `setup` function inside the cogs?